pom.xml

```diff
@@ -94,14 +94,14 @@
         <lombok.version>1.18.20</lombok.version>
         
         <postgresql.version>42.4.1</postgresql.version>
-        <opengauss.version>3.0.0</opengauss.version>
+        <opengauss.version>3.1.0-og</opengauss.version>
```

Comment:
What's the different between `3.1.0` and `3.1.0-og`?

Reply:
- I checked at https://github.com/oracle/graalvm-reachability-metadata/pull/168 and the difference is reflected in the package name. 

- `3.1.0-og` uses the same `org.opengauss.**` package name as `3.0.0`, while `3.1.0` uses the breakthrough `org.postgresql.**` as the package name, I can't find an explanation for this package name change from opengauss anywhere, it's ridiculous, I can only use `3.1.0-og`.

Reply:
Got it.